services/wallet/common/helpers.go

```diff
@@ -90,3 +90,16 @@ func UsernameToLabel(username string) [32]byte {
 
 	return label
 }
+
+// GetNameWrapperAddress returns the address of the ENS Name Wrapper contract for a given chain ID.
```

Comment:
we normally include an `addresses.go` file next to the contract wrapper for these kinds of functions.